nixos/modules/services/backup/duplicity.nix

```diff
@@ -100,20 +122,24 @@ in
 
         environment.HOME = stateDirectory;
 
-        serviceConfig = {
-          ExecStart = ''
-                        ${pkgs.duplicity}/bin/duplicity ${escapeShellArgs (
-                          [
-                            cfg.root
-                            cfg.targetUrl
-                            "--archive-dir"
-            stateDirectory
-                          ]
-                          ++ concatMap (p: [ "--include" p ]) cfg.include
-                          ++ concatMap (p: [ "--exclude" p ]) cfg.exclude
-                          ++ cfg.extraFlags
-            )}
+        script =
+          let
+            target = escapeShellArg cfg.targetUrl;
+            extra = escapeShellArgs ([ "--archive-dir" stateDirectory ] ++ cfg.extraFlags);
```

Comment:
Mayne rename 'extra' into 'flags'?

Reply:
I think we enter bikeshedding territory here.

Reply:
True. Feel free to ignore, I don't decide.
But it's confusing between extra an extraFlags. It's not extra, it's all flags. Or options.